### DISCUSSION TITLE
Invalid repository name, use the format owner/name.

### DISCUSSION THREAD

**Discussion by chrismcqueen-da** [community/community#45375] [GitHub Education]
This error has come up on the old discussion board, but adding a new thread here to see if there's something I can do to resolve the issue.

Here's a quick video showing what's happening: https://app.screencast.com/D5xWFiIQDZT3w

Essentially, I try to create a new assignment and get a cryptic (to me) error:

> Invalid repository name, use the format owner/name.

<img width="1392" alt="Screenshot 2023-01-25 at 2 45 04 PM" src="https://user-images.githubusercontent.com/116895554/214686320-6d7ea143-7f79-43e8-b92a-8062e28a5929.png">

**Comment by VladCuciureanu** [comment]
While I do not know the cause of the issue...
Some people reported that logging out and back in fixes the issue. Have you tried that?

**Reply by axelmagnus** [reply]
This did work for me. thanx!
